tests/integration-tests/tests/schedulers/conftest.py

```diff
@@ -119,46 +118,37 @@ def _create_database_stack(stack_factory, request, region, vpc_stack_for_databas
 
 
 @pytest.fixture(scope="class")
-def database_factory(request, vpc_stack_for_database):
-    created_database_stacks = defaultdict(dict)
+def database(request, vpc_stack_for_database, region):
     stack_factory = CfnStacksFactory(request.config.getoption("credential"))
 
-    logging.info("Setting up database_factory fixture")
-
-    def _database_factory(
-        existing_database_stack_name,
-        test_resources_dir,
-        region,
-    ):
-        if existing_database_stack_name:
-            logging.info("Using pre-existing database stack named %s", existing_database_stack_name)
-            return existing_database_stack_name
-
-        if not created_database_stacks.get(region, {}).get("default"):
-            logging.info("Creating default database stack")
-            database_stack = _create_database_stack(stack_factory, request, region, vpc_stack_for_database)
-            created_database_stacks[region]["default"] = database_stack.name
-
-        logging.info("Using database stack %s", created_database_stacks.get(region, {}).get("default"))
-        return created_database_stacks.get(region, {}).get("default")
-
-    yield _database_factory
-
-    for region, stack_dict in created_database_stacks.items():
```

Comment:
why remove the dictionary for this? Dont we need Region specific DB Stacks anymore?